docs/commands.md

```diff
@@ -425,6 +425,11 @@ react-native build-android --extra-params "-x lint -x test"
 
 Installs passed binary instead of building a fresh one. This command is not compatible with `--tasks`.
 
+
+#### `--userId` <number>
+
+Id of the User Profile you want to install the app on.
```

Comment:
since this feature is named "user profiles", shouldn't the flag be `--userProfile`? Id is quite ambiguous, but idk

Reply:
Or just `--user`, since this is how adb names it